src/test/resources/rest-api-spec/test/rescore/20_rescoring.yml

```diff
@@ -93,17 +152,27 @@ setup:
-  - match:  { hits.hits.2._source: { "body": "hello hello hello hello hello hello hello hello bye bye", "gender": "m"} }
-  - match:  { hits.hits.3._source: { "body": "hello hello hello hello hello hello hello bye bye bye", "gender": "m"} }
-  - match:  { hits.hits.4._source: { "body": "hello hello hello hello hello hello bye bye bye bye", "gender": "m"} }
-  - match:  { hits.hits.5._source: { "body": "hello hello hello hello hello bye bye bye bye bye", "gender": "f"} }
-  - match:  { hits.hits.6._source: { "body": "hello hello hello hello bye bye bye bye bye bye", "gender": "f"} }
-  - match:  { hits.hits.7._source: { "body": "hello hello hello bye bye bye bye bye bye bye", "gender": "f"} }
-  - match:  { hits.hits.8._source: { "body": "hello hello bye bye bye bye bye bye bye bye", "gender": "f"} }
-  - match:  { hits.hits.9._source: { "body": "hello bye bye bye bye bye bye bye bye bye", "gender": "f"} }
+  - length: { hits.hits: 20 }
+  - match:  { hits.hits.0._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello", "gender": "m" } }
+  - match:  { hits.hits.1._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello bye", "gender": "m"} }
+  - match:  { hits.hits.2._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello bye bye", "gender": "m"} }
+  - match:  { hits.hits.3._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello bye bye bye", "gender": "m"} }
+  - match:  { hits.hits.4._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello bye bye bye bye", "gender": "m"} }
+  - match:  { hits.hits.5._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello bye bye bye bye bye", "gender": "m"} }
+  - match:  { hits.hits.6._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello hello hello bye bye bye bye bye bye", "gender": "m"} }
+  - match:  { hits.hits.7._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello hello bye bye bye bye bye bye bye", "gender": "m"} }
+  - match:  { hits.hits.8._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello hello bye bye bye bye bye bye bye bye", "gender": "m"} }
+  - match:  { hits.hits.9._source: { "body": "hello hello hello hello hello hello hello hello hello hello hello bye bye bye bye bye bye bye bye bye", "gender": "m"} }
+  - match:  { hits.hits.10._source: { "body": "hello hello hello hello hello hello hello hello hello hello bye bye bye bye bye bye bye bye bye bye", "gender": "f"} }
```

Comment:
remove this results till the end of the list. I will add them later myself back.